docs/hub/advanced-compute-options.md

```diff
@@ -0,0 +1,66 @@
+# Advanced Compute Options
+
+Enterprise Hub organizations gain access to advanced compute options to accelerate their machine learning journey.
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
+</Tip>
+
+## Train on NVIDIA DGX Cloud
+
+Train on NVIDIA DGX Cloud offers a simple no-code training job creation experience powered by Hugging Face AutoTrain and Hugging Face Spaces. Instantly access NVIDIA GPUs and avoid the time-consuming work of writing, testing, and debugging training scripts for AI models.
+
+### How it works
+
+Read the [blogpost for Train on NVIDIA DGX Cloud](https://huggingface.co/blog/train-dgx-cloud#how-it-works).
+
+### Supported architectures
+
+#### Transformers
+
+| Architecture              |
+|---------------------------|
+| Llama                     |
+| Falcon                    |
+| Mistral                   |
+| Mixtral                   | 
+| T5                        |
+| gemma                     |
+
+#### Diffusers
+
+| Architecture              |
+|---------------------------|
+| Stable Diffusion          |
+| Stable Diffusion XL       |
+
+### Pricing
+
+Usage of Train on NVIDIA DGX Cloud is billed by the minute of the GPU instances used during your training jobs. Usage fees accrue to your Enterprise Hub Organizations’ current monthly billing cycle, once a job is completed. You can check your current and past usage at any time within the billing settings of your Enterprise Hub Organization.
+
+| NVIDIA GPU                | GPU Memory                | On-Demand Price/hr        |
+|---------------------------|---------------------------|---------------------------|
+| NVIDIA L40S               | 48GB                      |$2.75                      |
+| NVIDIA H100               | 80GB                      |$8.25                      |
+
+## Serverless Inference powered by NVIDIA
+
+Serverless Inference powered by NVIDIA offers access to NVIDIA AI technology GPUs an [NVIDIA Inference Microservices](https://www.nvidia.com/en-us/ai/) (NIM) in a serverless way. Use standardized APIs and a few lines of code to run inference in a pay-as-you-go pricing model.
```

Comment:
```suggestion
Serverless Inference powered by NVIDIA offers access to NVIDIA AI technology GPUs on [NVIDIA Inference Microservices](https://www.nvidia.com/en-us/ai/) (NIM) in a serverless way. Use standardized APIs and a few lines of code to run inference in a pay-as-you-go pricing model.
```

The line is a bit long, not sure if `on` is correct but `an` didn't sound right either.

Reply:
```suggestion
Serverless Inference powered by NVIDIA offers access to [NVIDIA Inference Microservices (NIM)](https://www.nvidia.com/en-us/ai/) powered by NVIDIA H100s in a serverless way. Use standardized APIs and a few lines of code to run inference in a pay-as-you-go pricing model.
```
Agreed, trying to clarify

Reply:
thank you for the feedback